src/Ant.cpp

```diff
@@ -2,26 +2,49 @@
 #include <antgame/Food.h>
 #include <antgame/World.h>
 #include <stdlib.h>
+#include <math.h>
 
 #include <easyloggingpp/easylogging++.h>
 
 #include <boost/geometry.hpp>
 
+
+#define PI 3.14159265
 namespace {
 
 float RandFloat(float lo, float hi) {
 	return lo + static_cast <float> (rand()) /( static_cast <float> (RAND_MAX/(hi - lo)));
 }
 
 static constexpr auto kStartingEnergy = 100;
+auto kStartingDirection = RandFloat(-180, 180);
 
 }
 
 Ant::Ant(const Point& initialPosition, const std::string& name)
 : Agent(initialPosition, name)
+, m_direction(kStartingDirection)
 , m_energy(kStartingEnergy)
 { }
 
+void Ant::Turn(float Angle) { //updates direction of Ant
+	m_direction += Angle;
+
+	// if (m_direction < -180){
```

Comment:
remove comments.

There should still be a way to enforce the direction to be in range `[-180, 180]`

Reply:
to do later